src/test/java/com/google/devtools/build/lib/remote/SimpleBlobStoreActionCacheTest.java

```diff
@@ -441,12 +446,16 @@ public void testUploadDirectoryNested() throws Exception {
     final Path barDir = execRoot.getRelative("bar");
 
     ActionResult.Builder result = ActionResult.newBuilder();
-    client.upload(result, null, null, null, execRoot, ImmutableList.<Path>of(barDir), false);
+    Action action = Action.newBuilder().build();
+    ActionKey actionKey = DIGEST_UTIL.computeActionKey(action);
+    Command cmd = Command.newBuilder().build();
```

Comment:
I'm wondering if there should be a helper for the above three lines. They appear a lot.

Reply:
yep. done

Reply:
done